Jenkinsfile

```diff
@@ -152,6 +152,12 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             """)
         }
 
+        stage('Test QEMU Image') {
```

Comment:
Also, super bikeshed: WDYT about having `Kola` in the stage name? E.g. `QEMU Kola Run`?